client/my-sites/site/index.jsx

```diff
@@ -58,6 +71,74 @@ module.exports = React.createClass( {
 		event.preventDefault(); // this doesn't actually do anything...
 	},
 
+	starSite: function() {
+		const site = sites.getSelectedSite();
```

Comment:
Why do we call `sites.getSelectedSite` here instead of using `this.props.site` ?